lisa/tools/whoami.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+
+
+class Whoami(Tool):
```

Comment:
put new tool classes into the `__init__.py`, so it's easy to reference.

Reply:
Add a method like `get_name` to return structured result, so it doesn't need to use like `.stdout.strip()` everytime.

Reply:
Is it added?

Reply:
This tool was added by Matthew so I removed the commit from this PR